src/main/java/com/redbrokers/processing/dto/SingleOrder.java

```diff
@@ -1,16 +1,38 @@
 package com.redbrokers.processing.dto;
 
+import com.fasterxml.jackson.annotation.JsonAlias;
 import com.redbrokers.processing.enums.Side;
 import com.redbrokers.processing.model.Execution;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 import java.math.BigDecimal;
+import java.util.List;
 
+@NoArgsConstructor
+@Getter
+@Setter
 public class SingleOrder {
```

Comment:
Am a bit confused though. What is the difference between an SingleOrder, and an Order

Reply:
a single order is from the orderbook.